examples/views/dynamic_user_dropdown_async.py

```diff
@@ -0,0 +1,94 @@
+import asyncio
+
+from asyncinit import asyncinit
```

Comment:
Examples should not require external dependencies, and an identical async example isn't necessary

Reply:
You don't need an external library for async init.


```python
class AsyncInit:
    async def __new__(cls, *args, **kwds):
        obj = super().__new__(cls)
        await obj.__init__(*args, **kwargs)
        return obj

    async def __init__(self, *args, **kwargs):
        await smth()
```

Reply:
> You don't need an external library for async init.
> 
> ```python
> class AsyncInit:
>     async def __new__(cls, *args, **kwds):
>         obj = super().__new__(cls)
>         await obj.__init__(*args, **kwargs)
>         return obj
> 
>     async def __init__(self, *args, **kwargs):
>         await smth()
> ```
> 
> This can be modified to subclass other stuff like `discord.ui.View` or a select menu, or even a Cog if you want.

It still isn't needed, and these examples are meant to remain mostly simple

Reply:
If the async example isnt needed, it can be removed entirely.  But, in regard to the examples intended to be simple, there should to be a source for more advanced configs.  Theres plenty of examples for similar effects in the JS libraries.  I always have a hard time finding Python examples for stuff like this.  I know there are people who are looking for it, as I saw plenty of people asking when I googled it.  I couldnt find a solution, so I worked through it, and made one.  My pull request is to save the next person from having to try to figure it out.

Reply:
Our examples are meant to be for new users.
You are free to create advanced examples and inform us so we can maybe add them to a link list

Reply:
That would be cool. Maybe have it in the guide?

Reply:
would be possible yes